scripts/cluster-install/rabbitmq_config.yaml

```diff
@@ -0,0 +1,22 @@
+rabbitmq:
+  username: cloudify
+  password: 'c10udify'
+
+  cluster_members:
+    <first rabbitmq instance-name>:
+      default: <first rabbitmq private-ip>
+    <second rabbitmq instance-name>:
+      default: <second rabbitmq private-ip>
+    <third rabbitmq instance-name>:
+      default: <third rabbitmq private-ip>
+
+  cert_path: '/home/centos/install_cluster/certs/<first rabbitmq instance-name>_cert.pem'
+  key_path: '/home/centos/install_cluster/certs/<first rabbitmq instance-name>_key.pem'
+  ca_path: '/home/centos/install_cluster/certs/ca.pem'
+
+  nodename: '<first rabbitmq instance-name>'
+
```

Comment:
I might've missed it earlier, but I don't think I saw a join_cluster config entry anywhere?

Reply:
It's in the function [`_prepare_rabbitmq_config` ](https://github.com/cloudify-cosmo/cloudify-dev/blob/install-cluster-script/scripts/cluster-install/install.py#L109)